Runtime/Behaviours/CinemachineStoryboard.cs

```diff
@@ -91,6 +91,18 @@ public enum FillStrategy
         [Tooltip("If checked, Camera transform will not be controlled by this virtual camera")]
         public bool m_MuteCamera;
 
+        /// <summary>
+        /// The render mode of the canvas on which the storyboard is drawn.
+        /// </summary>
+        [Tooltip("The render mode of the canvas on which the storyboard is drawn.")]
+        public RenderMode m_RenderMode = RenderMode.ScreenSpaceOverlay;
+        
+        /// <summary>
+        /// The render camera for the storyboard when render mode is not ScreenSpaceOverlay.
+        /// </summary>
+        [Tooltip("The render camera for storyboard when render mode is not ScreenSpaceOverlay.")]
+        public Camera m_RenderCamera;
```

Comment:
I am puzzled by this.  Normally, the camera is inferred from the Brain.  Why do we need a camera override here?

Reply:
I was not sure how story board is used. I can get the camera from brain too. This way there is no need for render camera. I'll fix it.